### PR TITLE
release: v0.8.0

### PR DESCRIPTION
**Pull request by iGormilhit**
* Updates the version number (rero-ils/version.py).
* Updates the RELEASE-NOTES.rst and the CHANGES.rst files.

Co-Authored-by: Igor Milhit <igor.milhit@rero.ch>

## Why are you opening this PR?

To publish the `v0.8.0` release.

## How to test?

Check changed files.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: RELEASE-NOTES.rst [hunk 1]
```diff
@@ -18,6 +18,176 @@
...
+      differentiate standard records (ie monograph), electronic records
+      or serials.
+   -  Updates the holdings JSON schema to:
+
+      -  Configure the holdings record editor, in which the publication
+         pattern will be defined.
+      -  Encode the publication pattern in the holdings records.
+
+   -  Computes predicted issues preview based on the publication pattern
+      of a given holdings.
```
Comment (pronguen) [inline RELEASE-NOTES.rst:126]:
```suggestion
      of a given holding.
```

File: RELEASE-NOTES.rst [hunk 2]
```diff
@@ -18,6 +18,176 @@
...
+   budget.
+-  Adds a route to update the pickup location of an existing request.
+
+Fixtures
+~~~~~~~~
+
+-  Regenerates documents and holdings after the changes for the serials
+   predictions.
+-  Adds in the fixtures the 10 publication patterns that are most used
+   in actual libraries of the RERO network.
```
Comment (pronguen) [inline RELEASE-NOTES.rst:146]:
```suggestion
-  Adds in the fixtures the 10 publication patterns that are currently most used
   in the RERO network.
```

File: RELEASE-NOTES.rst [hunk 3]
```diff
@@ -18,6 +18,176 @@
...
+Acquisition
+~~~~~~~~~~~
+
+-  Begins to implement serials management with the creation of a manual
+   prediction:
+
+   -  Adds an ``holdings_type`` parameter to the holdings to
+      differentiate standard records (ie monograph), electronic records
+      or serials.
+   -  Updates the holdings JSON schema to:
```
[the remark this answers is not in the document]
Comment (iGormilhit) [reply RELEASE-NOTES.rst:119]: The problem is that "holdings" stands for "holdings record", as I understand it. The LoC documentation do not use "holding" in the singular form. I still make the error, though. Maybe It should be "the JSON schema of the holdings records".

File: RELEASE-NOTES.rst [hunk 4]
```diff
@@ -18,6 +18,176 @@
...
+~~~~~~~
+
+-  Allows the ``setup`` script to succeed even if records don’t have any
+   ``responsibilityStatement``.
+
+Instance
+~~~~~~~~
+
+-  Updates Invenio framework to version ``3.2.1``!
+-  Upgrades ``https-proxy-agent`` for security reason.
```
Comment (mmo) [inline RELEASE-NOTES.rst:168]: for security reasons.

File: RELEASE-NOTES.rst [hunk 5]
```diff
@@ -18,6 +18,176 @@
...
+-  `#788 <https://github.com/rero/rero-ils/issues/778>`__: The search
+   suggestions are displayed only after a click in the input field, even
+   after 3 typed characters.
+-  `#939 <https://github.com/rero/rero-ils/issues/939>`__: The email
+   notification should use the email of the library as sender.
+-  `#960 <https://github.com/rero/rero-ils/issues/960>`__: A librarian
+   using the web browser Chrome/Chromium wasn’t able to update the
+   pickup location of a request.
+-  `rero/rero-ils-ui#140 <https://github.com/rero/rero-ils-ui/issues/140>`__:
+   the library facet is missing in the professional interface.
```
Comment (mmo) [inline RELEASE-NOTES.rst:189]: is missing -> was missing (?)

**Pull request merged**